metricshub-doc/src/site/markdown/release-notes.md

```diff
@@ -5,6 +5,81 @@ description: Learn more about the new features, changes and improvements, and bu
 
 <!-- MACRO{toc|fromDepth=1|toDepth=1|id=toc} -->
 
+## MetricsHub Enterprise Edition v3.0.01
+
+### MetricsHub Enterprise Edition v3.0.01
+
+#### Changes and Improvements
+
+| ID                                                                          | Description                                                 |
+| --------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| [**\#98**](https://github.com/MetricsHub/metricshub-enterprise/issues/98)   | Added a `user` CLI to manage REST API users                 |
+| [**\#100**](https://github.com/MetricsHub/metricshub-enterprise/issues/100) | Upgraded OpenTelemetry Collector Contrib to version 0.136.0 |
+
+### MetricsHub Enterprise Connectors v109
+
+#### What's New
+
+| ID                                                                        | Description                                                                                      |
+| ------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| [**\#30**](https://github.com/MetricsHub/enterprise-connectors/issues/30) | Added support for [**AIX**](./connectors/aix.html) systems via command lines                     |
+| [**\#70**](https://github.com/MetricsHub/enterprise-connectors/issues/70) | Added support for [**EMC Isilon**](./connectors/emcisilonrest.html) storage systems via REST API |
+
+#### Changes and Improvements
+
+| ID                                                                        | Description                                                                                                                                                                                                                                                                |
+| ------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [**\#72**](https://github.com/MetricsHub/enterprise-connectors/issues/72) | The `storage.size` metric is now collected for storage systems and physical disks by the following storage array connectors: <ul><li>**Dell EMC PowerMax Storage (REST)**</li><li>**Pure Storage FA Series (REST)**</li><li>**Pure Storage FA Series v2 (REST)**</li></ul> |
```

Comment:
@NassimBtk , I suggest adding a link to the connector page if users want to know more about these connectors.